sbt-converter/src/main/scala/org/scalablytyped/converter/internal/PluginRemoteCache.scala

```diff
@@ -59,7 +120,8 @@ object PluginRemoteCache {
       ec:          ExecutionContext,
   ): Int =
     remoteCache match {
-      case RemoteCache.Disabled => 0
+      case RemoteCache.Disabled       => 0
+      case RemoteCache.S3(_, pullUri) => 0
       case RemoteCache.Rsync(_, pullUri) =>
         val localRunFile = runCacheKey.path(cacheDir)
```

Comment:
here you can extract everything after `case RemoteCache.Rsync(_, pullUri) =>` into a `def` and just pass in the `pullUri`, I guess everything else could be the same.

Once you have both sides implemented you can test all this locally by running `stPublish` and moving/deleting `~/.cache/scalablytyped/runs` and `~/.ivy2/local/org.scalablytyped`. If everything works you'll just see downloads and no conversion process